tools/make-srpm

```diff
@@ -39,29 +39,24 @@ else
 fi
 
 tmpdir=$(mktemp -d $PWD/srpm-build.XXXXXX)
-tar xf "$source" -O '*/tools/cockpit.spec' > $tmpdir/cockpit.spec
+
+rpmbuild_args=(
+    --define "_topdir $tmpdir"
+    --define "_srcrpmdir $(pwd)"
+)
+
+rpmbuild "${rpmbuild_args[@]}" -ts ${source} | sed -n 's@^Wrote:.*/@@p'
 
 # on RHEL/CentOS 8 (only), cockpit is delivered as two mostly identical source packages.
 # "cockpit" with build_basic=1, and "cockpit-appstream" with build_optional=1
 # the spec has proper build_* defaults depending on the Name:
-if grep -qE '^ID=.*(rhel|centos)' /etc/os-release && grep -qE '^VERSION_ID="8' /etc/os-release; then
-    sed '/^Name:/ s/$/-appstream/' < $tmpdir/cockpit.spec > $tmpdir/cockpit-appstream.spec
-fi
+if grep -q 'platform:el8' /lib/os-release; then
```

Comment:
We usually use /etc/os-release, but indeed that has never been an ideal place. So agree to using the "static" path. However, the [canonical location](https://www.freedesktop.org/software/systemd/man/os-release.html) is /usr/lib/os-release.

Reply:
Right.  Of course.  I'll fix that to `/usr`.